sos/report/__init__.py

```diff
@@ -1297,16 +1297,21 @@ def _collect_plugin(self, plugin):
                 pool.shutdown(wait=True)
                 pool._threads.clear()
         if self.opts.estimate_only:
-            from pathlib import Path
-            tmpdir_path = Path(self.archive.get_tmp_dir())
-            self.estimated_plugsizes[plugin[1]] = sum(
-                    [f.lstat().st_size for f in tmpdir_path.glob('**/*')])
+            # call "du -s -B1" for the tmp dir to get the disk usage of the
+            # data collected by the plugin - if the command fails, count with 0
+            tmpdir = self.archive.get_tmp_dir()
+            try:
+                du = _plug.exec_cmd('du -sB1 %s' % tmpdir)
```

Comment:
Rather than running this in a plugin, I think we should just directly use `sos_get_command_output()` here.